pkgs/os-specific/darwin/apple-sdk-11.0/default.nix

```diff
@@ -56,6 +56,21 @@ let
     '';
   };
 
+  mkClangStdenv = stdenv: let
```

Comment:
Isn't the fact that this is Clang kind of superfluous, as it just uses the cc from the provided stdenv?

Reply:
We can use another version of clang ?

Reply:
I don't understand your question, but to clarify, I'm talking about solely the function name here.

Reply:
Renamed to `mkStdenv`. Feel free to suggest a better (or more endomorphic?) name.